### PR TITLE
Added company iCount

### PR DESCRIPTION
**Pull request by leon2589**

### CODE REVIEW

File: companies/513947119.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "iCount",
+    "description": "Everything a business needs and more. Finally, a system that can meet all of your business’s needs from the basic to the most advanced.",
```
Comment (mluggy) [inline companies/513947119.json:3]: Consider something less obscure. I.e "Accounting, billing, time management and more for small-medium businesses.

File: companies/513947119.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "categoryId": 10,
+    "size": "s",
+    "logoFile": "logos/icount_logo.webp",
```
Comment (mluggy) [inline companies/513947119.json:6]: add a 200x200px webp, png or jpeg file as part of your submission

**Review by mluggy — CHANGES_REQUESTED** [review]
see 2 comments

File: companies/513947119.json (hunk 2 above)
Comment (leon2589) [reply companies/513947119.json:6]: I already uploaded, maybe you missed it?
image

**Review by mluggy — APPROVED** [review]

**Pull request merged**